snapcraft/internal/meta.py

```diff
@@ -371,7 +371,11 @@ def _wrap_apps(self, apps):
             if os.path.splitext(f)[1] == '.desktop':
                 os.remove(os.path.join(gui_dir, f))
         for app in apps:
-            self._wrap_app(app, apps[app])
+            wrapper = "legacy"
+            if "wrapper" in apps[app]:
+                wrapper = apps[app]["wrapper"]
```

Comment:
`wrapper = apps[app].get("wrapper", legacy)` is shorter

Not sure if snapcraft uses anything that can derive the default from the schema.